Mathlib/Analysis/SpecialFunctions/Gaussian/GaussianIntegral.lean

```diff
@@ -351,3 +352,15 @@ theorem Complex.Gamma_one_half_eq : Complex.Gamma (1 / 2) = (π : ℂ) ^ (1 / 2
   convert congr_arg ((↑) : ℝ → ℂ) Real.Gamma_one_half_eq
   · simpa only [one_div, ofReal_inv, ofReal_ofNat] using Gamma_ofReal (1 / 2)
   · rw [sqrt_eq_rpow, ofReal_cpow pi_pos.le, ofReal_div, ofReal_ofNat, ofReal_one]
+
+open scoped Nat in
+/-- The special-value formula `Γ(k + 1 + 1/2) = (2 * k + 1)‼ * √π / (2 ^ (k + 1))` for half-integer
+values of the gamma function in terms of `Nat.doubleFactorial`. -/
+lemma Real.Gamma_nat_add_one_add_half (k : ℕ) :
+    Gamma (k + 1 + 1 / 2) = (2 * k + 1 : ℕ)‼ * √π / (2 ^ (k + 1)) := by
```

Comment:
I'm guessing you wanted to avoid nat subtraction?
```suggestion
    Gamma (k + 1 / 2) = (2 * k - 1 : ℕ)‼ * √π / (2 ^ k) := by
```
Perhaps it's worth having both?

Reply:
It's fine to have both, so I've added your version, but note that the `k = 0` version is a bit weird because it's taking advantage of the junk values for truncated subtraction (i.e., using `2 * 0 - 1 = 0`). The version I wrote is the "morally correct" one. This is the off-by-one nature of the Gamma function.